src/ai/backend/client/cli/vfolder.py

```diff
@@ -74,7 +74,10 @@ def list_allowed_types():
 @click.argument('host', type=str, default=None)
 @click.option('-g', '--group', metavar='GROUP', type=str, default=None,
               help='Group ID or NAME. Specify this option if you want to create a group folder.')
-def create(name, host, group):
+@click.option('--host-path', type=bool, is_flag=True,
```

Comment:
:+1: for making as a flag option.

However, I think it's better to keep "unmanaged" as the terminology.
This also adds a nuance: the user becomes aware of that she/he needs to know what she/he is doing.
Let's rename this option to `--unmanaged`.